cpp/SConstruct

```diff
@@ -71,14 +83,7 @@ carfac_sources = [
     ]
 carfac = env.Library(target = 'carfac', source = carfac_sources)
 Default(carfac)
-
-env.Command('tmp/libgtest.a', [],
-            [
-                Delete('tmp'),
-                Copy('tmp', os.environ['GTEST_SOURCE']),
-                'cd tmp && cmake . && make',
-            ])
-
+# #8 rely on users gtest install into /usr/lib instead of making a copy to tmp
```

Comment:
What is this `#8`? And did I completely miss this question for my first review? :)